boards/xtensa/esp32/common/Kconfig

```diff
@@ -77,10 +77,6 @@ config SSD1680_GPIO_PIN_CS
 	int "Pin that select the chip on SPI bus"
 	default "5" if ARCH_BOARD_TTGO_T5V2_ESP32
 
-config SSD1680_GPIO_PIN_DTA_CMD
```

Comment:
Please, extend this change to the other pins as well.
They are specific to `ARCH_BOARD_TTGO_T5V2_ESP32`, so they should not be placed under the common Kconfig.

`SSD1680_GPIO_PIN_CS` should instead use `ESP32_SPI3_CSPIN` option for the CS pin.